config.xml

```diff
@@ -42,6 +42,32 @@
     <splash density="port-xxhdpi" src="resources/android/splash/drawable-port-xxhdpi-screen.png" />
     <splash density="port-xxxhdpi" src="resources/android/splash/drawable-port-xxxhdpi-screen.png" />
   </platform>
+  <platform name="ios">
+    <edit-config file="*-Info.plist" mode="merge" target="NSCameraUsageDescription">
+      <string>Used to scan barcodes</string>
+    </edit-config>
+    <edit-config file="*-Info.plist" mode="merge" target="NSMicrophoneUsageDescription">
+      <string>Used for audio input tasks</string>
+    </edit-config>
+    <preference name="Suppresses3DTouchGesture" value="false" />
+    <preference name="SuppressesLongPressGesture" value="false" />
```

Comment:
Why did we disable longPress?

Reply:
Sorry I was testing something and forgot to remove it. Let me remove it.